### ISSUE TITLE
Predictions of the model 

### ISSUE THREAD

**Issue by shah25** [karimrattani/rare_event_detection#8]
![image](https://user-images.githubusercontent.com/25223499/53028440-ded3d080-348c-11e9-8eb3-958c9f9ccb53.png)

every prediction is 1, hardly there is any prediction as 0 . I have  tried many combinations for epochs and number of videos , but the accuracy is around 60 percent and also the cost is above 1. The cost never reaches 0.0x . Also please could you please explain me the concept of bags in yr code, when i executed your code in the while loop of the file algorithm.py (create bags) function, it only ran once that is random.choice (X) it selects only one video for training is what I understood from the code.
Could u please help me out with it?

PFA:
![image](https://user-images.githubusercontent.com/25223499/53028877-c0baa000-348d-11e9-8d3c-d40a6838cb33.png)

**Comment by hemaprabha** [comment]
@shah25  Hi 
Can u please tell me the steps  to run the pre-trained model?

**Comment by karimrattani** [comment]
@shah25 Are you getting those prediction using a pretrained model provided in Readme or did you train it and tried to use it?
For bags concept, please refer to the paper https://arxiv.org/pdf/1801.04264.pdf
I've also uploaded Software Design Document in the Repo.
